src/graphs.jl

```diff
@@ -266,6 +266,72 @@ function edgifyrates(rxs, specs)
     es
 end
 
+# create edges from one reaction complex to another reaction complex
+function edgifycomplex(δ)
+    attr = Attributes()
+    return map(δ) do p
+        return Edge([p[1], p[2]] , attr)
+    end
+end
+
+# modify vector of string of complexes into graphviz compatible strings
+function modifystrcomp(strcomp::Vector{String})
+  for i in 1:length(strcomp)
+    #   cannot allow + as per graphviz
+      if occursin("+",strcomp[i])
+          strcomp[i] = replace(strcomp[i], "+" => "₊");
```

Comment:
Maybe use `\circ` or a dash `-`? I think the subscript plus is a bit strange.

Reply:
> Maybe use `\circ` or a dash `-`? I think the subscript plus is a bit strange.

I will try `\circ`.
I had used subscript plus as its unicode since simple `+` was not allowed, which would have been closest resemblance to how we write in `@reaction_network

Reply:
This too is resolved !, Now we can use the default `+`